osquery/filesystem/filesystem.cpp

```diff
@@ -147,6 +148,175 @@ Status listFilesInDirectory(const boost::filesystem::path& path,
   }
 }
 
+Status listDirectoriesInDirectory(const boost::filesystem::path& path,
```

Comment:
This should probably be in `<osquery/filesystem.h>`, like `listFilesInDirectory`